etl/steps/data/meadow/idmc/2024-08-02/internal_displacement.meta.yml

```diff
@@ -0,0 +1,163 @@
+  processing_notes:
+    population_share: |-
+      The share of internally displaced people of the total population is calculated as the number of internally displaced people in a country divided by the total population of the country, multiplied by 100. For this calculation we use our long-term population data set, which can be explored in the [population data explorer](https://ourworldindata.org/explorers/population-and-demography).
+
+
+dataset:
+  update_period_days: 365
+
+tables:
+  internal_displacement:
+    variables:
+      total_stock_displacement:
+        title: Internally displaced people
+        description_short: |-
+          [Internally displaced people](#dod:internally-displaced) due to both conflict and violence and natural disaster. This includes all people who are currently displaced, no matter which year they originally got displaced.
+        unit: people
+        short_unit: ""
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+          - {definitions.idmc_definitions.disaster}
+        description_processing: This figure is the sum of the number of people displaced due to conflict and violence and the number of people displaced due to natural disasters.
+      share_of_internally_displaced_pop:
+        title: Share of internally displaced people of total population
+        description_short: |-
+          The share of [internally displaced people](#dod:internally-displaced) of all people living in the country. This includes all people who are currently displaced, due to both conflict and violence and disaster and no matter which year they originally got displaced.
+        unit: "%"
+        short_unit: "%"
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+          - {definitions.idmc_definitions.disaster}
+        processing_level: major
+        description_processing: |-
+          - {definitions.processing_notes.population_share}
+          - This figure is the sum of the number of people displaced due to conflict and violence and the number of people displaced due to natural disasters.
+      conflict_stock_displacement:
+        title: Internally displaced people due to conflict
+        description_short: |-
+          [Internally displaced people](#dod:internally-displaced) due to conflict. This includes all people who are currently displaced due to conflict, no matter which year they originally got displaced.
+        unit: people
+        short_unit: ""
+        description_from_producer: |-
+          - {definitions.idmc_definitions.internally_displaced_people}
+          - {definitions.idmc_definitions.conflict}
+      share_of_conflict_displaced_pop:
+        title: Share of internally displaced people due to conflict of total population
```

Comment:
same here - maybe "Internally displaced people due to conflict as a share of total population" but depends if you want to use these as the actual titles for charts. If not then you don't really need to worry about it and the titles of the variables make sense